com.unity.render-pipelines.high-definition/Documentation~/master-stack-unlit.md

```diff
@@ -111,6 +111,7 @@ Depending on the [Graph Settings](#graph-settings) you use, Shader Graph can add
 [!include[](snippets/shader-properties/surface-options/alpha-to-mask.md)]
 [!include[](snippets/shader-properties/surface-options/double-sided.md)]
 [!include[](snippets/shader-properties/surface-options/ss-depth-offset.md)]
+[!include[](snippets/shader-properties/surface-options/conservative-depth-offset.md)]
```

Comment:
we don't have conservative depth for unlit iirc

Reply:
If we have depth offsets on unlit shader, i don't see why we couldn't have the conservative constraint for them aswell

Reply:
ah, indeed - sorry made a confusion - unlit.shader don't have a need for it as we don't allow to control depth offset there BUT unlit master node have it. Is the current PR handling this case?